test/src/test/java/org/corfudb/infrastructure/logreplication/LogReplicationConfigManagerTest.java

```diff
@@ -137,24 +107,10 @@ public void testConfigGeneration() {
     public void testConfigUpdate() throws Exception {
         LogReplicationConfigManager configManager = new LogReplicationConfigManager(runtime);
         verifyExpectedConfigGenerated(configManager.getConfig());
```

Comment:
Shouldn't we also verify the streamsToReplicate?

Reply:
Nice catch. Addressing it